src/utils/fetch-node-data.ts

```diff
@@ -120,3 +120,42 @@ function convertMsToDHM(ms: number): string {
 
   return `${daysString}${hoursString}${minutesString}`;
 }
+
+export function canUnstake(
```

Comment:
when looking at the business logic here: https://github.com/shardeum/shardeum/pull/172/files
we need to implement the same rules for `canUnstake`. so we also have to check syncingTimestamp, rewardStart and rewardEnd